### PR TITLE
frontend: circular progress bar / tx status to use progress bar

### PR DESCRIPTION
**Pull request by originalblend337**
- circular progress bar component
- transactions (including details dialog) to use circular progress bar instead of indicator dots
- remove unused css from transactions

### CODE REVIEW

File: frontends/web/src/components/progressRing/progressRing.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            className={[style.container, className ? className : ''].join(' ')}
+            width={width}
+            height={width}
+            viewBox={`0 0 ${width} ${width}`}>
+            <circle
+                className={style.background}
+                cx={width / 2}
+                cy={width / 2}
+                r={radius}
+                strokeWidth={2} />
```
Comment (Tomasvrba) [inline frontends/web/src/components/progressRing/progressRing.tsx]: I tested this PR on a low res screen and the circles were a bit hard to see, especially the gray/pending (Elvis will complain I'm sure :D)
I tried changing the strokeWidth to 3 and it looks quite nice. What do you think?

File: frontends/web/src/components/progressRing/progressRing.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+const ProgressRing = ({
+    className,
+    generic,
+    isComplete,
+    isError,
+    value,
+    width,
+}: RenderableProps<ProgressRingProps>) => {
+    const radius = (width - 2) / 2;
```
Comment (Tomasvrba) [inline frontends/web/src/components/progressRing/progressRing.tsx]:
```suggestion
    const radius = (width - 3) / 2;
```
see below

File: frontends/web/src/components/progressRing/progressRing.tsx (hunk 1 above)
Comment (Tomasvrba) [inline frontends/web/src/components/progressRing/progressRing.tsx]:
```suggestion
                strokeWidth={3} />
```

File: frontends/web/src/components/progressRing/progressRing.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                className={[
+                    style.foreground,
+                    !generic && isComplete ? style.complete : style.pending,
+                    generic && style.generic,
+                    isError && style.error,
+                ].join(' ')}
+                cx={width / 2}
+                cy={width / 2}
+                r={radius}
+                strokeWidth={2}
```
Comment (Tomasvrba) [inline frontends/web/src/components/progressRing/progressRing.tsx]:
```suggestion
                strokeWidth={3}
```

File: frontends/web/src/components/transactions/transaction.jsx [hunk 4]
```diff
@@ -111,26 +113,22 @@ export default class Transaction extends Component {
...
         const sDate = time ? this.parseTimeShort(time) : '---';
-        const statusStyle = {
-            complete: style.statusIndicatorComplete,
-            pending: style.statusIndicatorPending,
-            failed: style.statusIndicatorFailed,
-        }[status];
         const statusText = {
             complete: 'Complete',
-            pending: 'Pending',
+            pending: `Pending`,
```
Comment (Tomasvrba) [inline frontends/web/src/components/transactions/transaction.jsx]: While you're at it making changes here, let's move all three of these to the i18n file?

**Comment by Tomasvrba** [comment]
Sorry it took a while to review :)
Maybe something to consider for a new PR or adding as on implementation to https://github.com/digitalbitbox/bitbox-wallet-app/pull/736 
We could use the tooltip to show the number of confirmations when hovering over the progress circle of a transaction. That way you can quickly see the M/N tz conf numbers without opening the whole tx details dialog

File: frontends/web/src/components/progressRing/progressRing.tsx (hunk 1 above)
Comment (originalblend337) [reply frontends/web/src/components/progressRing/progressRing.tsx]: sure, let's give it a shot and we can adjust if any other opinions/better ideas come up. thanks

**Pull request merged**